mungegithub/mungers/submit-queue.go

```diff
@@ -647,6 +657,34 @@ func (sq *SubmitQueue) getQueueStatus() []byte {
 	return sq.marshal(status)
 }
 
+// getMetrics returns a json representation of important stats about the
+// submit queue's health, for external monitoring/graphing/alerting.
+func (sq *SubmitQueue) getMetrics() []byte {
+	sq.Lock()
+	defer sq.Unlock()
+
+	status := sq.e2e.GetBuildStatus()
+	blocked := false
+	for _, state := range status {
+		if state.Status != "Stable" {
+			blocked = true
+		}
+	}
```

Comment:
I think this is wrong. You really want some combination of 
`sq.e2e.GCSBasedStable()` and `sq.e2e.GCSWeakStable()`
I believe that after @lavalamp changes this is a bit more nuanced.  Ignore `sq.e2e.Stable()` as that's dead and needs cleaned up...